src/App.jsx

```diff
@@ -23,10 +23,7 @@ function App() {
         <NewFirstScreen />
       ) : (
         <div>
-          <header>Test Test</header>
           <MainMenu />
-
-          <footer></footer>
         </div>
       )}
     </div>
```

Comment:
This comment is about line 40/37: Why do you use emotion here? I would recommend to stick to styled-components

Reply:
I forgot to delete it - sorry. That was content from the tutorial I've watched :)